server/test/api/debtorGroups.js

```diff
@@ -83,6 +119,28 @@ describe('The /debtor_groups HTTP API ENDPOINT', function () {
     .catch(helpers.handler);
   });
 
+  it('POST /debtor_groups/ create a new debtor group (convention)', function () {
+    return agent.post('/debtor_groups/')
+    .send(conventionGroup)
+    .then(function (res) {
+      expect(res).to.have.status(201);
+      expect(res.body.id).to.exist;
+      expect(res.body.id).to.be.equal(conventionGroup.uuid);
+    })
+    .catch(helpers.handler);
+  });
+
+  it('POST /debtor_groups/ create a new debtor group (locked convention)', function () {
+    return agent.post('/debtor_groups/')
+    .send(lockedConventionGroup)
+    .then(function (res) {
+      expect(res).to.have.status(201);
+      expect(res.body.id).to.exist;
+      expect(res.body.id).to.be.equal(lockedConventionGroup.uuid);
```

Comment:
It would be good check to make sure the convention is locked with an additional GET request (since that is what your test says happens).